src/com/oltpbenchmark/DBWorkload.java

```diff
@@ -649,25 +649,22 @@ private static void PrintLatencies(List<Worker> workers, boolean outputVerboseRe
     if (outputVerboseRes) {
       resultOut = new StringBuilder();
       resultOut.append("\n");
-      resultOut.append("=====================================FAILURE LATENCIES====================================\n");
-      resultOut.append(" Transaction |  Count   | Percentage | Avg. Latency | P99 Latency | Connection Acq Latency\n");
+      resultOut.append("==============================FAILURE LATENCIES==============================\n");
+      resultOut.append(" Transaction |  Count   | Avg. Latency | P99 Latency | Connection Acq Latency\n");
       for (int i = 0; i < list_failure_latencies.size(); ++i) {
         String op = transactionTypes.get(i + 1);
         List<Integer> latencies = list_failure_latencies.get(i);
         List<Integer> conn_latencies = list_failure_conn_latencies.get(i);
-        double percentage = 100.0 * latencies.size() / (latencies.size() + list_latencies.get(i).size());
         resultOut.append(String.format(
-                "%12s |%9s |%10.2f%% |%13.2f |%12.2f |%23.2f\n",
-                op, latencies.size(), percentage, getAverageLatency(latencies), getP99Latency(latencies),
```

Comment:
why remove percentage? seems very useful

Reply:
If a transaction failed multiple times (can happen in a retry case), each failed attempt would be accounted once here and hence the percentage of failures would increase. In actuality, we only want to count of failures per retry attempt and this information is available in the retry attempts section